clusterDeployer.py

```diff
@@ -63,7 +66,6 @@ def __init__(self, cc: ClustersConfig, ai: AssistedClientAutomation, steps: list
         self._client: Optional[K8sClient] = None
         self._ai = ai
         self._secrets_path = secrets_path
-        self._bf_iso_path = "/root/iso"
```

Comment:
global state is harder than local state. CDA used to have almost no global state/constants. At most this should be 

`self._bf_iso_path = _DEFAULT_BF_ISO_PATH`

Reply:
`def __init__(self, cc: ClustersConfig, ai: AssistedClientAutomation, steps: list[str], secrets_path: str):`

If we replace that by

`def __init__(self, cc: ClustersConfig, ai: AssistedClientAutomation, steps: list[str], secrets_path: str, iso_path: str = _DEFAULT_BF_ISO_PATH):`

then I'm ok with this change.

Reply:
`self._bf_iso_path` is currently an attribute of `ClusterDeployer`. However, it never changes, it's always the same constant value. It never gets initialize to anything else, and never gets reset.

There is no reason to add a constructor argument "iso_path". This does not need to be configurable or be held in `self._bf_iso_path`.

This is a constant, and the patch turns this into a constant.

We have dozens of such constants throughout the code (`git grep "^[A-Z_]\+ = ['\"]"`), this is not different.